roles/inspec/defaults/main.yml

```diff
@@ -9,9 +9,9 @@ inspec:
   dependency_path: /etc/inspec/
   profiles:
     openstack_security:
-      name: inspec-openstack-security
+      name: openstack-baseline
```

Comment:
maybe we could add a `profiles_to_purge` alongside `profiles` ?
Then we can remove the old and add the new.